src/components/UserPortal/Login/Login.tsx

```diff
@@ -44,16 +44,12 @@ export default function login(props: InterfaceLoginProps): JSX.Element {
           },
         });
 
-        if (data) {
-          localStorage.setItem('token', data.login.accessToken);
-          localStorage.setItem('userId', data.login.user._id);
-
-          navigator.clipboard.writeText('');
-          /* istanbul ignore next */
-          window.location.assign('/user/organizations');
-        } else {
-          toast.warn(t('notAuthorised'));
```

Comment:
It looks like you got the code coverage up by removing functionality. If this is son, it is not in the spirit of our community and the desire to have a better product. @noman2002 Please verify.

Reply:
Please write code that tests the code base without removing functionality